app/main.py

```diff
@@ -1,8 +1,20 @@
 class Person:
-    # write your code here
-    pass
+    people = dict()
+
+    def __init__(self, name: str, age: int) -> None:
+        self.name = name
+        self.age = age
 
 
 def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+    list_ = list()
+    for person in people:
+        list_.append(Person(person["name"], person["age"]))
+        Person.people[person["name"]] = list_[-1]
+    for i, person in enumerate(people):
```

Comment:
i is a bad name for variable use full names for naming, f.e index or person_index